sentry/organization_integrations.go

```diff
@@ -25,7 +25,7 @@ type OrganizationIntegration struct {
 	Icon        string                          `json:"icon"`
 	DomainName  string                          `json:"domainName"`
 	AccountType string                          `json:"accountType"`
-	Scopes      *string                         `json:"scopes"`
+	Scopes      *[]string                       `json:"scopes"`
```

Comment:
Why it has to be `*[]string` instead of `[]string`? `[]string` itself is a pointer and can have the `nil` value.

Reply:
Whoops! Fixed in https://github.com/Canva/go-sentry/pull/6/commits/fbe3df19e32aa605c224683aa03521951330ed60

Reply:
Fast fix :-D 🚀

Reply:
😆 😄